src/coreComponents/integrationTests/fluidFlowTests/testCompFlowUtils.hpp

```diff
@@ -80,25 +80,10 @@ void setupProblemFromXML( ProblemManager & problemManager, char const * const xm
   dataRepository::Group & commandLine =
     problemManager.getGroup< dataRepository::Group >( problemManager.groupKeys.commandLine );
 
-  commandLine.registerWrapper< integer >( problemManager.viewKeys.xPartitionsOverride.key() ).
+  commandLine.getWrapper< integer >( problemManager.viewKeys.xPartitionsOverride.key() ).
     setApplyDefaultValue( mpiSize );
 
-  xmlWrapper::xmlNode xmlProblemNode = xmlDocument.getChild( dataRepository::keys::ProblemManager );
-  problemManager.processInputFileRecursive( xmlDocument, xmlProblemNode );
-
-  DomainPartition & domain = problemManager.getDomainPartition();
-
-  constitutive::ConstitutiveManager & constitutiveManager = domain.getConstitutiveManager();
-  xmlWrapper::xmlNode topLevelNode = xmlProblemNode.child( constitutiveManager.getName().c_str());
-  constitutiveManager.processInputFileRecursive( xmlDocument, topLevelNode );
-
-  MeshManager & meshManager = problemManager.getGroup< MeshManager >( problemManager.groupKeys.meshManager );
-  meshManager.generateMeshLevels( domain );
-
-  ElementRegionManager & elementManager = domain.getMeshBody( 0 ).getBaseDiscretization().getElemManager();
-  topLevelNode = xmlProblemNode.child( elementManager.getName().c_str());
-  elementManager.processInputFileRecursive( xmlDocument, topLevelNode );
```

Comment:
This was just duplicating the contents of  `problemManager.parseXMLDocument( xmlDocument );`.